frontend/src/component/FlashMessage.styl

```diff
@@ -38,6 +38,9 @@
         &__paragraph
           padding-right 20px
 
+#connection_error
```

Comment:
avoid using html ids for styling, prefer class usage
The idea behind it is to keep specificity low and manageable